kstreams-live-update/README.md

```diff
@@ -3,19 +3,18 @@
 This demo shows how to use KStreams to join two CDC event streams created by Debezium,
 do some calculation on the joined stream and push the merged events to a client using WebSockets.
 
-The domain is that of weather stations that measure temperature data.
-There's an application _event-source_, which persists random temperature measurements.
-In reality this could for instance expose a REST API, to which individual sensors post their events.
+The domain is that of orders that belong to given categories.
+There's an application _event-source_, which persists random orders.
 
 The application has two tables:
 
-* `stations`: Named weather stations
-* `temperature_measurements`: temperature measurements for a station with a value and timestamp
+* `categories`: Product categories
+* `orders`: orders with a given category and a (random) sales prices
 
 Debezium is used to capture changes to the two tables in the application's underlying MySQL database.
 
-The _temperature-aggregator_ application runs KStreams to join measurements with stations,
-group the events by station name and calculate the average temperature per station.
+The _aggregator_ application runs KStreams to join orders with categories,
+group the events by category name and calculate the average sales price per category.
```

Comment:
Sum all all sales per category might have a better sense in this context?

Reply:
Ah yes, that makes sense I'd say. I'll change it.